### PR TITLE
refacto about us

### PR DESCRIPTION
**Pull request by LuckyShuii**

### CODE REVIEW

**Review by Monost0ne — APPROVED** [review]
putain c'est beau

File: assets/styles/about_us.scss [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+}
+
+article a img {
+    margin-right: 1rem;
+}
+
+/* Mobile */
+
+@media screen and (max-width:1100px) {
+    main {
```
Comment (Benja-san) [inline assets/styles/about_us.scss:200]: is it a global behavior for all main of all pages ? shift it into global css file

File: assets/styles/about_us.scss [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+    .longSeparation {
+        display: none;
+    }
+
+    .smallSeparation {
+        display: flex;
+    }
+}
+
+@-webkit-keyframes pulse {
```
Comment (Benja-san) [inline assets/styles/about_us.scss:275]: if this pulse keyframe is used anywhere else than in this file, shift it into global declarations

**Review by Benja-san — APPROVED** [review]
Read commentaries before merge, and make changes if the coments are relevant

**Pull request merged**